core/src/documentation/ide-vscode-setup.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Spark source code in Visual Studio Code IDE</title>
+    <style>
+
+        body {
+            background-color: #d3d6d9;
+        }
+        header {
+            background-color: #e7eaee;
+            border: 1px solid #cccccc;
+            margin-top: 10px;
+            padding-left: 20px;
+            border-bottom: 20px solid #335588;
+        }
+        header h1 {
+            font-size: 1.7em;
+            font-weight: bold;
+            color: #670e15;
+        }
+        header p {
+            color: black;
+        }
+        article {
+            background-color: #fff;
+            border: 1px solid #999;
+            padding: 40px;
+            margin: 30px;
+        }
+        img {
+            height: 250px;
+        }
+    </style>
+</head>
+<body>
+    <article>
+        <header>
+            <h1>Spark source code in Visual Studio Code IDE</h1>
+            <p>
+                This is a short tutorial on how to load the Spark source code in the Visual Studio Code IDE.
+            </p>
+        </header>
+        <p>
+            Visual Studio Code or VSCode is a fast editor and ships with great editing features. It includes support for debugging,
```

Comment:
It's a minor thing, but i like to be consistent with naming. I think official short name is with the space "VS Code". Would be great to use same naming through out the guide.